packages/manager/modules/pci/src/projects/project/data-processing/submit-job/submit-job.html

```diff
@@ -77,8 +79,15 @@
         ng-if="$ctrl.submitRetries>0 && !$ctrl.isSubmitting"
     >
         <span class="oui-icon oui-icon-error_circle"></span>
-        <span class="submitjob__errormessage"
+        <span
+            class="submitjob__errormessage"
+            ng-if="$ctrl.badRequestErrorMessage.length == 0"
             >{{ ::'data_processing_submit_job_error_label' | translate}}</span
         >
+        <span
+            class="submitjob__errormessage"
+            ng-if="$ctrl.badRequestErrorMessage.length > 0"
```

Comment:
In order to keep consistency:
```suggestion
            data-ng-if="$ctrl.badRequestErrorMessage.length > 0"
```

Reply:
Commit amended with correction